### ISSUE TITLE
Set passcode not enforced when creator/moderator leaves room

### ISSUE THREAD

**Issue by stephendonner** [jitsi/jitsi-meet#9932]
### Description:

Set passcode not enforced when creator/moderator leaves room

### Steps to reproduce:

1. load `beta.talk.brave.com` using Brave (release version)
2. click on the `Start free 1:1 call` button
3. enter your name
4. click on the `Join meeting` button
5. click on the `...` on the bottom, black toolbar and choose `Security options`
6. click `Add passcode`
7. add a passcode of your choosing
8. click on `Add`
9. copy the meeting URL
10. close the current meeting/call's tab
11. open the meeting URL in another browser
12. try to join the call

### Expected behavior:

All users joining the call should be required to enter the passcode

### Actual behavior:

No passcode is prompted for; users are able to join at will

**Comment by saghul** [comment]
Did you leave the call in between? Once a room is empty it's effectively destroyed, including its configuration like the password.

**Comment by stephendonner** [comment]
> Did you leave the call in between? Once a room is empty it's effectively destroyed, including its configuration like the password.

Yes, that's step 10.  Sorin pointed us towards using the `SETTINGS_PROVISIONING` from https://developer.8x8.com/jaas/docs/webhooks-payload#settings_provisioning

**Comment by saghul** [comment]
Exactly. Feel free to close this once you test that,
